### DISCUSSION TITLE
Select component: Auto-highlight first option as user types?

### DISCUSSION THREAD

**Discussion by ericwilhite** [mantinedev/mantine#4041] [Q&A] [answered]
I'm writing a point-of-sale application where users are repeatedly processing through the same form. This form has ~10 Select components with search enabled.

Ideally as the user types / searches, the first item in the dropdown would be highlighted so they can filter to a match and just hit enter. Instead, they have to arrow down to select the first item.

Over time the keystrokes add up. 

Is there a way to do this now that I'm just missing? Or can anyone point in the right direction to implement such functionality?

**Comment by rtivital** [comment, marked as answer]
Use `hoverOnSearchChange` prop

**Reply by ericwilhite** [reply]
I must have looked at the props list 100 times and somehow did not see that option. 

Thank you for this awesome library and all the work you put into it!
